nav2_map_server/include/nav2_map_server/map_saver.hpp

```diff
@@ -13,105 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef NAV2_MAP_SERVER__MAP_SAVER_HPP_
-#define NAV2_MAP_SERVER__MAP_SAVER_HPP_
-
-#include <string>
-#include <memory>
-
-#include "rclcpp/rclcpp.hpp"
-#include "nav2_util/lifecycle_node.hpp"
-#include "nav2_msgs/srv/save_map.hpp"
-
-#include "map_io.hpp"
-
-namespace nav2_map_server
-{
-
-/**
- * @class nav2_map_server::MapSaver
- * @brief A class that provides map saving methods and services
+/*
+ * File combining all the definitions of map_saver of different type.
```

Comment:
Remove this file - update the appropriate locations to import only the appropriate 2D/3D version as required. I know this was better for development, but the objects using the 2D libraries won't use the 3D (and vise versa) so including this file would increase the binary sizes of the servers by alot for 2D if they have to also include PCL.